css/style.css

```diff
@@ -0,0 +1,827 @@
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  position: fixed;
+  z-index: 2;
+  top: 0;
+  background-color: #fefefe;
+}
+
+.mid-div-left {
+  display: none;
+}
+
+.top-elements {
+  display: none;
+}
```

Comment:
[Required] It is not a good practice to repeat styling. Let's follow the DRY principle and make our files clean.